src/controller/message.ts

```diff
@@ -30,4 +35,19 @@ export class MessageController {
   ): Promise<Result.Result<RESTError | Error, RESTGetAPIChannelMessageResult>> {
     return this.fetchMessage.fetch(channelId, messageId);
   }
+
+  /**
+   * Create a message in a channel.
+   * @param channelId Channel ID to send message
+   * @param body Message contents, embeds, etc.
+   * @returns success: {@link RESTGetAPIChannelMessageResult} or error: {@link RESTError} or {@link Error}
```

Comment:
```suggestion
   * @returns success: {@link RESTGetAPIChannelMessageResult}, error: {@link RESTError} or {@link Error}
```
orが違う意味で2回使われているのでこういう感じ？